Task7/Task7/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task7
+{
+    //Каждое из чисел m и n нечетное. 
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите два числа");
+ 
+            int m = int.Parse(Console.ReadLine());
+            int n = int.Parse(Console.ReadLine());
+            Console.WriteLine(EvenOdd(m, n));
+
+        }
+        private static string EvenOdd (int m, int n)
```

Comment:
Имя метода не совсем понятное. Лучше начинать его с глагола в императиве. И чтобы смысл, **что** делает метод, был ясен. Например, так
```suggestion
        private static string IsBothNumbersOdd (int m, int n)
```